pkg/ctl/create/cluster.go

```diff
@@ -107,6 +107,10 @@ func doCreateCluster(cmd *cmdutils.Cmd, ngFilter *filter.NodeGroupFilter, params
 	cfg := cmd.ClusterConfig
 	meta := cmd.ClusterConfig.Metadata
 
+	if meta.Name != "" && !cmdutils.IsValidNameArg(meta.Name) {
+		return cmdutils.ErrInvalidName(meta.Name)
+	}
+
```

Comment:
you can create nodegroups as part of create cluster. do we want to verify they are valid too?

Reply:
added validation checks for multiple nodegroups 👍🏻